app/Image/FileDeleter.php

```diff
@@ -146,7 +148,9 @@ public function do(): void
 				// Laravel and Flysystem does not support symbolic links.
 				// So we must use low-level methods here.
 				if (is_link($absolutePath) || file_exists($absolutePath)) {
-					if (!unlink($absolutePath)) {
+					try {
+						\Safe\unlink($absolutePath);
+					} catch (\Throwable) {
 						$firstException = $firstException ?: new \RuntimeException('unlink failed: ' . $absolutePath);
```

Comment:
```suggestion
					try {
						\Safe\unlink($absolutePath);
					} catch (\Throwable $e) {
						$firstException = $firstException ?: $e;
```

See previous comment.

Reply:
Fixed it myself in https://github.com/LycheeOrg/Lychee/pull/1336/commits/7e9a9ec414515da7c30bc166b412c7bbdf4ccfe7.